card.js

```diff
@@ -194,17 +213,25 @@
               event.preventDefault();
               return;
             case 39: // Right
-            case 40: // Down
               // Move focus forward
               self.trigger('next');
               event.preventDefault();
               return;
+            case 40: // Down
```

Comment:
I think we should avoid using `event.which` now it is deprecated.

Reply:
@otacke I was not refactoring the old code. `which` was there before.